Description: Make w make better guesses for user process
 Based on suggestion by Herbert Xu <herbert@gondor.apana.org.au>
Bug-Debian: http://bugs.debian.org/187808
Author: Craig Small <csmall@debian.org>
--- a/w.c
+++ b/w.c
@@ -171,7 +171,7 @@
 	    secondbest = tmp;
 	}
 	if(!ignoreuser && uid != tmp->euid && uid != tmp->ruid) continue;
-	if(tmp->tgid != tmp->tpgid) continue;
+	if(tmp->pgrp != tmp->tpgid) continue;
 	if(best && tmp->start_time <= best->start_time) continue;
     	best = tmp;
     }
